speechbrain/nnet/losses.py

```diff
@@ -31,7 +33,7 @@ def transducer_loss(
     target_lens,
     blank_index,
     reduction="mean",
-    use_torchaudio=True,
+    framework="torchaudio",
```

Comment:
I see it is coming from this change. Suggesting there are more frameworks. Please check with `grep -r use_torchaudio recipes` if you found all references and adopted the change.

( I'm surprised it are only 10 lines in the entire recipes folder ! )

Reply:
yup, that's all recipes there were used. Just fixed their README files.